lib/node_modules/@stdlib/math/base/special/roundsd/manifest.json

```diff
@@ -0,0 +1,102 @@
+{
+    "options": {
```

Comment:
This file should be dedented by two spaces to ensure consistent 2-space indentation.